src/diffpy/morph/morphs/morphfuncy.py

```diff
@@ -1,9 +1,61 @@
+"""class MorphFuncy -- apply a user-supplied python function to the y-axis."""
+
 from diffpy.morph.morphs.morph import LABEL_GR, LABEL_RA, Morph
 
 
 class MorphFuncy(Morph):
-    """Apply the user-supplied Python function to the y-coordinates of the
-    morph data"""
+    """General morph function that applies a user-supplied function to the
+    y-coordinates of morph data to make it align with a target.
+
+    Configuration Variables
+    -----------------------
+    function: callable
+        The user-supplied function that applies a transformation to the
+        y-coordinates of the data.
+
+    parameters: dict
+        A dictionary of parameters to pass to the function.
+        These parameters are unpacked using **kwargs.
```

Comment:
This line not needed.  User doesn't care how the program works, just how to use it.